src/analyzer/analyzerkey.cpp

```diff
@@ -5,7 +5,7 @@
 
 #include "analyzer/constants.h"
 #include "analyzer/plugins/analyzerqueenmarykey.h"
-#include "proto/keys.pb.h"
```

Comment:
The changes altering cpp code should be done in a separate PR. I am happy to merge them primarily.

Reply:
I moved some code into `src/proto/CMakeLists.txt`, which makes this change unnecessary.

Reply:
https://github.com/mixxxdj/mixxx/pull/2280/commits/b86e683b4b1871cd4c1966811f3eebf600df220a